lib/assertions.ts

```diff
@@ -172,9 +194,10 @@ export default class DOMAssertions {
    * assert.dom('input[type="text"]').isRequired();
    *
    * @see {@link #isNotRequired}
+   * {@link https://www.w3.org/TR/wai-aria-1.1/#aria-required}
    */
-  isRequired(message?: string): DOMAssertions {
-    isRequired.call(this, message);
+  isRequired(message?: string, options?: { withAriaSupport?: boolean }): DOMAssertions {
+    isRequired.call(this, message, options);
```

Comment:
I think this is missing the fallback to `this.options.withAriaSupport` if the option is not explicitly specified in the passed in `options` argument

Reply:
hmm, looks like you've moved that fallback into the functions. I think I'd prefer it if the functions would always get an `options` object with a `withAriaSupport` value and don't have to depend on `this` except for `this.pushResult()`. 🤔